docs/cpp/source/check-doxygen.sh

```diff
@@ -30,7 +32,7 @@ python tools/setup_helpers/generate_code.py                 \
 popd
 
 # Run doxygen and log all output.
-doxygen 2> original-doxygen-log.txt
+doxygen $(dirname $0) 2> original-doxygen-log.txt
```

Comment:
You should quote this shell interpolation

Reply:
amend line 35
>>doxygen "$(dirname $0)" 2> original-doxygen-log.txt
quote could prevent variable separate, thanks a lot